Planet Express/src/PlanetExpress.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+
+public class PlanetExpress {
+	private int thirst;
+	private int work;
+	private int horde;
+	private int hunger;
+	private int receipts;
+	private boolean stable;
+	
+
+	public PlanetExpress() {
+		thirst = 50;
+		work = 50;
+		horde = 50;
+		hunger = 50;
+		receipts = 50;
+		stable = true;
+	}
+
+	public int getHunger() {
+		return this.hunger;
+	}
+	
+	public int getThirst() {
+		return this.thirst;
+	}
+	
+	public int getWork() {
+		return this.work;
+	}
+	
+	public int getHorde() {
+		return this.horde;
+	}
+	 
+	public int getReceipts() {
+		return this.receipts;
+	}
+	
+	private boolean isStable() {
+		if (this.hunger >= 100 || this.thirst >= 100 || this.work >= 100) {
+			this.stable = false;
+		} else if (this.receipts <= 0 || this.horde <= 0) {
+			this.stable = false;
+		}
+		return this.stable;
+	}
+	
+	public void act(String action) {
+		switch (action) {
```

Comment:
Nice use of the case statement here. You should always have a default for every case statement though, in case the input is not what is expected.